lib/Jsrt/ChakraCommon.h

```diff
@@ -1040,9 +1040,9 @@ typedef unsigned char* ChakraBytePtr;
     /// </returns>
     CHAKRA_API
         JsParseScript(
-            _In_z_ const wchar_t *script,
+            _In_z_ const CSTR_TYPE *script,
```

Comment:
If we have a set of Jsrt API we are no longer planning to support on xplat, IMO we should mark them separately. This PR does not intend to mark what we may or may not support on xplat.

For the CSTR_TYPE macro. See the comment on its definition. We should warn the user that we do not use same type cross platform. Either that particular method is supported or not.